src/pages/projects/esg/Index.vue

```diff
@@ -0,0 +1,358 @@
+<template>
+    <component :is="currentLayout" class="index">
+        <p v-html="$page.main.content" class="mb"></p>
+
+        <h2 class="mb-3">
+            {{ $page.datasetObjectives.heading }}
+        </h2>
+
+        <div class="info-4 mb">
+            <div v-for="(objective, i) in $page.datasetObjectives.objectives" :key="i" class="card blue-card shadow-sm">
+                <h3 class="card-header mt-0">
+                    {{ objective.title }}
+                </h3>
+                <div class="card-body p-2">
+                    <p>{{ objective.content }}</p>
+                </div>
+            </div>
+        </div>
+
+        <EsgInteractivePosterVue class="mb" />
+
+        <h2>
+            {{ $page.datasetLinks.heading }}
+        </h2>
+
+        <div class="external-links mb">
+            <b-button
+                v-for="(link, i) in $page.datasetLinks.external_links"
+                class="border-0"
+                :key="i"
+                :title="link.name"
+                :href="link.url"
+                variant="outline-secondary"
+            >
+                <div class="mx-auto">
+                    <g-image :src="link.image" :alt="link.name" />
+                </div>
+            </b-button>
+        </div>
+
+        <h2 class="mb-3">
+            {{ $page.datasetExpectedResults.heading }}
+        </h2>
+
+        <div class="info-4 mb">
+            <div
+                v-for="(result, i) in $page.datasetExpectedResults.expected_results"
+                :key="i"
+                class="card blue-card shadow-sm"
+            >
+                <h3 class="card-header mt-0">
+                    {{ result.title }}
+                </h3>
+                <div class="card-body p-2">
+                    <p>{{ result.content }}</p>
+                </div>
+            </div>
+        </div>
+
+        <p v-html="$page.section_1.content" class="mb"></p>
+
+        <div class="news-and-events mb">
+            <div class="card blue-card shadow-sm">
+                <h2 class="card-header mt-0">
+                    <span class="icon fas fa-bullhorn mr-1" />
+                    News
+                </h2>
+                <div class="card-body p-2">
+                    <div>
+                        <ItemListBrief v-for="edge in $page.news.edges" :key="edge.id" :item="edge.node" />
+                    </div>
+                    <a :href="esgUrl('/projects/esg/news')" class="btn btn-secondary border-0 btn-sm w-75">
+                        <b>More News</b>
```

Comment:
I can't decide if I like this button more than the other approach in the rest of the hub where the header is the link, but we should probably pick one and standardize for these itemlist components.